lumen/sources/intake.py

```diff
@@ -0,0 +1,57 @@
+import intake
+import numpy as np
+import param
+
+from ..util import get_dataframe_schema
+from .base import Source
+
+
+class IntakeSource(Source):
+    """
+    An IntakeSource loads data from an Intake catalog.
+    """
+
+    catalog = param.String(doc="""
+        The name of the catalogue, must be provided.""")
```

Comment:
```suggestion
        The name of the catalog; must be provided.""")
```

(Either "catalog" or "catalogue" is fine, but whatever it is should be the same in the docs for a single class, and I selected "catalog" since that's what Intake itself calls it.)